frontend/src/layout/navigation/ProjectNotice.tsx

```diff
@@ -143,6 +147,16 @@ export function ProjectNotice(): JSX.Element | null {
                 children: 'Reload page',
             },
         },
+        region_blocked: {
+            message:
+                'PostHog is not available in your region due to legal restrictions and will soon be blocked. Please contact support if you believe this is a mistake.',
```

Comment:
Is this for any user in a certain region, or will the org be blocked? It's not clear here. 

```suggestion
                'PostHog is not available in your region due to legal restrictions. People in restricted regions will soon be blocked from accessing PostHog. Please contact support if you believe this is a mistake.',
```

Reply:
I wonder if we should include a more specific reason, i.e., "You have been blocked because we have detected you are accessing PostHog from Russia which is a restricted region"